src/utilities/form-helper.ts

```diff
@@ -0,0 +1,72 @@
+import * as _ from 'lodash';
+import 'joi-extract-type';
+import * as FormData from 'form-data';
+import logger from './logger';
+
+interface UnmergeStrategyOptions {
+    key: string;
+    array: Array<string | Blob>;
+    form: FormData;
+}
+
+type UnmergeStrategyFunction = ({ key, array, form }: { key: string; array: Array<string | Blob>; form: FormData }) => void;
+
+export const unmergeStrategies = {
+    // unmerge strategies for arrays contained in objects -> form-data
+    // arrays should be included in the form-data with unique keys for OpenLab
+    unmergeUniqueKeysByIndex: ({ key, array, form }: UnmergeStrategyOptions): void => {
+        array.forEach((value: string | Blob, index: number) => {
+            form.append(`${key}${index + 1}`, value);
+        });
+    },
+    // WeBWorK answer submission expects duplicated keys (for multiple-select problems)
+    unmergeDuplicatingKey: ({ key, array, form }: UnmergeStrategyOptions): void => {
+        array.forEach((value: string | Blob) => {
+            form.append(key, value);
+        });
+    },
+    unmergeJoinArray: (joinChar: string) => ({ key, array, form }: UnmergeStrategyOptions): void => {
+        form.append(key, array.join(joinChar));
+    }
+};
+
+class FormHelper {
+    /**
+     * This function converts objects into FormData
+     * @param object the object to be converted
+     * @param unmerge *optional* the subroutine tasked with handling arrays inside `object`
+     * @returns the FormData converted from `object`
+     */
+    objectToFormData = ({
+        object,
+        unmerge,
```

Comment:
I think we should make unmerge a required property
At very least it should have a default value
```TypeScript
unmerge = unmergeStrategies.unmergeDuplicatingKey
```

I guess I can see it being optional if you think your object won't have any arrays 🤔

Reply:
exactly - there are several instances in the renderer helper where we have a clear grasp on what form-data we're sending and it doesn't require a merge strategy.